src/ucp/core/ucp_context.c

```diff
@@ -243,6 +243,11 @@ static ucs_config_field_t ucp_config_table[] = {
    "Maximal number of devices on which a rendezvous operation may be executed in parallel",
    ucs_offsetof(ucp_config_t, ctx.max_rndv_lanes), UCS_CONFIG_TYPE_UINT},
 
+  {"MIN_RNDV_CHUNK_SIZE", "16kb",
```

Comment:
minor: in other sizes in kb used "16k" format

Reply:
ok